src/test/java/org/dependencytrack/resources/v1/ProjectResourceTest.java

```diff
@@ -478,6 +507,102 @@ public void createProjectEmptyTest() {
         Assert.assertEquals(400, response.getStatus(), 0);
     }
 
+    @Test
+    public void createProjectWithExistingTeamRequiredTest() {
+        getUserToken(false);
+        Team AllowedTeam = qm.createTeam("AllowedTeam", false);
+        Project project = new Project();
+        project.setName("ProjectWithExistingTeamRequired");
+        qm.addUserToTeam(testUser, AllowedTeam);
+        qm.createConfigProperty("access-management", "acl.enabled", "true", IConfigProperty.PropertyType.BOOLEAN, "");
```

Comment:
Consider using constants for creating config properties, as seen in other places in this file:

```java
qm.createConfigProperty(
        ConfigPropertyConstants.ACCESS_MANAGEMENT_ACL_ENABLED.getGroupName(),
        ConfigPropertyConstants.ACCESS_MANAGEMENT_ACL_ENABLED.getPropertyName(),
        "true",
        ConfigPropertyConstants.ACCESS_MANAGEMENT_ACL_ENABLED.getPropertyType(),
        null
);
```